packages/menu/src/presenters/stylesheet.js

```diff
@@ -20,8 +20,10 @@ export default function stylesheet(props, themeData) {
     isPressed,
     role,
     selected,
+    unselect,
     stylesheet: customStylesheet
   } = props;
+  const unselectPropValue = unselect === undefined ? true : unselect;
```

Comment:
see comment above. if you set `unselect: true` in `defaultProps` you can then just check for the `unselect` prop.